1주차/10814/10814_python_파랑.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+count = int(sys.stdin.readline())
+users = [sys.stdin.readline() for _ in range(count)]
```

Comment:
```suggestion
import sys
readLine = sys.stdin.readline
count = int(readLine())
users = [readLine() for _ in range(count)]
```

위와 같이 함수 호출을 커스텀할 수도 있어요!ㅎ

Reply:
두 번 이상 호출될 때는 커스텀하는 게 편하겠네요ㅎㅎ